src/test/java/io/jenkins/plugins/appcenter/FreestyleTest.java

```diff
@@ -8,6 +8,7 @@
 import hudson.model.Node;
 import hudson.model.Result;
 import hudson.slaves.RetentionStrategy;
+import io.jenkins.plugins.appcenter.releasenotes.NoReleaseNotes;
```

Comment:
I think along with the release notes you should provide tests for the various release notes types and assert that the correct JSON is received by the mockwebserver.

Reply:
Yeah, thanks for your review. I will make a new PR with all these remarks ASAP.